### ISSUE TITLE
Another couple of missing recipes

### ISSUE THREAD

**Issue by fluffle** [TheLimePixel/GregBlock#113]
Hi Pixel,

Sorry for the stream of bug reports :-)

I've added the following recipe which appears to be missing for nuclearcraft (wrapped for readability): 

    assembler.recipeBuilder()
        .inputs([<ore:plateSteel>*4,<ore:plateTough>*4])
        .outputs([<nuclearcraft:cooler>])
        .property("circuit", 8)
        .duration(50)
        .EUt(16)
        .buildAndRegister();

Also the recipes for graphite and beryllium blocks (the two moderators nuclearcraft recognises) use nuclearcraft ingots in the compressor. The alloy smelter and extruder work fine.

    compressor.recipeBuilder()
        .inputs(<gregtech:meta_item_1:10204>*9)
        .outputs(<gregtech:compressed_9:5>)
        .duration(400)
        .EUt(2)
        .buildAndRegister();
    compressor.recipeBuilder()
        .inputs(<gregtech:meta_item_1:10007>*9)
        .outputs(<gregtech:compressed_0:5>)
        .duration(400)
        .EUt(2)
        .buildAndRegister();

Trying to remove the other recipes with e.g. `compressor.findReciipe(1, [<nuclearcraft:ingot:8>*9], null).remove();` hits an NPE -- I suspect it means the nuclearcraft based recipe couldn't be found :-/

Cheers,
--fluffle

**Comment by TheLimePixel** [comment]
The GT Blocks works as moderators.

**Comment by fluffle** [comment]
Yes, they do, it uses oredicts to find them. But you can't compress gt
graphite or beryllium ingots to gt blocks!
